### ISSUE TITLE
[BUG] Stubborn user blocking and strange overlay issues 

### ISSUE THREAD

**Issue by aidik** [vitorpamplona/amethyst#1087]
**Describe the bug**
When a WoT blocked profile is unblocked Amethyst blocks it again almost immediately. When this happen a strange note residue is left in the feed. Please see the recording. 

**To Reproduce**
Go to a profile blocked by the WoT feature. In my case `npub10sa7ya5uwmhv6mrwyunkwgkl4cxc45spsff9x3fp2wuspy7yze2qr5zx5p` and try to unblock it. 

**Expected behavior**
User unblocking will have permanent effect and would take precedence over WoT. 

**Video and Screenshots**
https://otherstuff.shaving.kiwi/52a58e9e83f171cddf0971ae4f6d7e98693d84c3f02af8cb189ec825614e7a05.mp4

**Device (please complete the following information):**
 - Phone Brand/Model: OnePlus 9 Pro
 - Android Version: 14
 - App Version: v0.91.0-play

**Comment by vitorpamplona** [comment]
It's not a WOT feature, it's the anti-spam feature. That account has 5+ completely equal posts in a short time and is classified as a spammer. You can turn the anti-spam filter off in the security filters on the left drawer OR you can follow the account.

**Comment by aidik** [comment]
Thank you for your answer. I can't get rid of the feeling that the behavior is strange and user action should take precedence over an automatic mechanic, but that is an opinion and not a bug. :-) 

The overly residue to the other hand seems like a real bug.

**Comment by vitorpamplona** [comment]
Yeah, we should add custom code to deal with this case on the unblock button. We just haven't had the time yet. :)

**Comment by git-3000** [comment]
Oh weird stuff like that has been happening to me too. I disabled the spam settings and the weird things stopped. 
I like having control of everything , I myself want to decide if it's a spam profile or not.

Now I understand what was happening
